index.bs

```diff
@@ -640,16 +669,42 @@ To <dfn for="SanitizerConfig">remove unsafe</dfn> from a |configuration|, do thi
 <div algorithm>
 To <dfn for="Sanitizer">set a configuration</dfn>, given a [=dictionary=] |configuration|, a [=boolean=] |allowCommentsAndDataAttributes|, and a {{Sanitizer}} |sanitizer|:
 
-1. [=list/iterate|For each=] |element| of |configuration|["{{SanitizerConfig/elements}}"] do:
-  1. Call [=allow an element=] with |element| and |sanitizer|'s [=Sanitizer/configuration=].
-1. [=list/iterate|For each=] |element| of |configuration|["{{SanitizerConfig/removeElements}}"] do:
-  1. Call [=remove an element=] with |element| and |sanitizer|'s [=Sanitizer/configuration=].
-1. [=list/iterate|For each=] |element| of |configuration|["{{SanitizerConfig/replaceWithChildrenElements}}"] do:
-  1. Call [=replace an element with its children=] with |element| and |sanitizer|'s [=Sanitizer/configuration=].
-1. [=list/iterate|For each=] |attribute| of |configuration|["{{SanitizerConfig/attributes}}"] do:
-  1. Call [=allow an attribute=] with |attribute| and |sanitizer|'s [=Sanitizer/configuration=].
-1. [=list/iterate|For each=] |attribute| of |configuration|["{{SanitizerConfig/removeAttributes}}"] do:
-  1. Call [=Sanitizer/remove an attribute=] with |attribute| and |sanitizer|'s [=Sanitizer/configuration=].
+1. If |configuration|["{{SanitizerConfig/elements}}"] [=map/exists=]:
+  1. Set |sanitizer|'s |configuration|["{{SanitizerConfig/elements}}"]
+      to &laquo;[]&raquo;
+  1. [=list/iterate|For each=] |element| of
+      |configuration|["{{SanitizerConfig/elements}}"]:
+    1. Call [=allow an element=] with |element| and
+        |sanitizer|'s [=Sanitizer/configuration=].
+1. If |configuration|["{{SanitizerConfig/removeElements}}"] [=map/exists=]:
```

Comment:
We shouldn't need the exists checks anymore, now that they are always initialized to empty lists.